### PR TITLE
1220-documentation-api-altdriver-update-description-and-parameters-with-connecttimeout-and-appname

### PR DESCRIPTION
**Pull request by SandyEma9**

### CODE REVIEW

**Comment by SandyEma9** [comment]
@robert-altom @RobertPoienar @gomboskriszta15 @iulianatodoran @MihaiCuc98 @ReichEdvin

File: Docs/source/pages/commands.md [hunk 1]
```diff
@@ -6,15 +6,17 @@ If you are looking for information on a specific function, class or method, this
...
-| port          | int     | No       | The default value is 13000.                                                           |
-| enableLogging | boolean | No       | The default value is false.                                                           |
+| Name           | Type    | Required | Description                                                                           |
+| -------------- | ------- | -------- | ------------------------------------------------------------------------------------- |
+| host           | string  | No       | The IP or hostname AltTester Unity SDK is listening on. The default value is "127.0.0.1". |
+| port           | int     | No       | The default value is 13000.                                                           |
+| enableLogging  | boolean | No       | The default value is false.                                                           |
+| connectTimeout | int     | No       | The connect timeout in seconds.The default value is 60                                |
+| appName        | boolean | No       | The name of the Unity application.The default value is `__default__`                  |
```
Comment (iulianatodoran) [inline Docs/source/pages/commands.md:19]: I think `appName` is of string type

File: Docs/source/pages/commands.md [hunk 2]
```diff
@@ -6,15 +6,17 @@ If you are looking for information on a specific function, class or method, this
 
 The **AltDriver** class represents the main app driver component. When you instantiate an AltDriver in your tests, you can use it to "drive" your app like one of your users would, by interacting with all the app objects, their properties and methods.
 
-An AltDriver instance will connect to the running instrumented Unity application. In the constructor, we need to tell the driver where (on what IP and on what port) the instrumented Unity App is running. We can also set some more advanced parameters, as shown in the table below:
+An AltDriver instance will connect to the running instrumented Unity application. In the constructor, we need to tell the driver where (on what IP and on what port) the instrumented Unity App with a specific name is running and for how many seconds to let the communication opened. We can also set some more advanced parameters, as shown in the table below:
```
Comment (iulianatodoran) [inline Docs/source/pages/commands.md:9]: The last sentence seems unnecessary as all parameters are already mentioned before, therefore no other advanced parameters remain to be described.

**Review by iulianatodoran — APPROVED** [review]
👍

**Review by robert-altom — APPROVED** [review]

**Pull request merged**